tools/manifestfile.py

```diff
@@ -632,6 +633,8 @@ def main():
         exit(1)
 
     m = ManifestFile(mode, path_vars)
+    for lib in args.add_library:
+        m.add_library(lib, os.path.join("$(MPY_LIB_DIR)", lib))
```

Comment:
I'm not sure it should prefix it with `MPY_LIB_DIR`.  What if the user wants a completely different path?

Reply:
Have changed it so it takes a path instead.